docs/about/index.rst

```diff
@@ -17,13 +41,174 @@ Code of conduct
+classification models&mdash; a setting that was commonly studied in the
+machine learning community.
+The paper and the Python package were well received, so Miro Dudik and Hanna
+Wallach sought to translate the research into an industry context.
+However, they discovered that practitioners typically need to address more
+fundamental fairness issues before they can use specific algorithms, and that
+mitigating unfairness in binary classification models is a relatively rare use
+case.
+They also discovered that fairness assessment is a common need, along with
+access to domain-specific guides to fairness metrics and unfairness mitigation
+algorithms.
+Additionally, many use cases take the form of regression or ranking, rather
+than classification.
+As a result of these insights, fairness assessment and use-case notebooks
+became key components of Fairlearn.
+Fairlearn also focuses on machine learning tasks beyond binary classification.
+
+The project was greatly expanded in the second half of 2019 thanks to the
+involvement of many contributors from Azure ML and Microsoft Research.
+At that time, the project started to have regular releases.
+
 .. _roadmap:
 
 Project roadmap
 ---------------
 
+As an open-source project, Fairlearn strives to incorporate the best of
+research and practice.
+AI is a rapidly evolving field, and fairness in AI is all the more so.
+We therefore encourage researchers, practitioners, and other stakeholders to
+contribute fairness metrics, unfairness mitigation algorithms, visualization
+capabilities, as well as educational materials to Fairlearn as we experiment,
+learn, and evolve the project together.
+
+There are many areas for future enhancement and growth. For example, Fairlearn
+currently supports only *group fairness*&mdash;i.e., fairness with respect to
+groups of people, such as those defined in terms of race, sex, age, or
+disability status&mdash;and not other conceptualizations of fairness, such as
+*individual fairness* or *counterfactual fairness.* Fairlearn also currently
+includes only a limited set of unfairness mitigation algorithms, although we
+note that these algorithms are not restricted to classification tasks. Besides
+adding fairness metrics, conceptualizations of fairness, and unfairness
+mitigation algorithms, we also hope that Fairlearn will expand to cover more
+complex machine learning tasks in areas like counterfactual reasoning,
+computer vision, and natural language processing.
+We also anticipate integrating Fairlearn with interpretability tools, such as
+`InterpretML <https://github.com/interpretml>`_.
+
+Ultimately, we hope that Fairlearn will become more than a software
+tool&mdash;a vibrant community and resource center that provides not only
+code, but also resources like domain-specific guides for when and when not to
+use different fairness metrics and unfairness mitigation algorithms.
+
+Short-term roadmap
```

Comment:
I'm afraid that listing the tasks here will incur lot of maintenance cost. Can it be just a link to the Github issues?

Reply:
That’s unfortunately super hard to figure out. There will always be 50 or more (likely way more) items. We can figure out some way of tagging but that’s not known to new contributors. High-level efforts like the ones listed here can remove themselves with the last PR from this guide. I don’t think that’s too bad.